packages/react-components/react-progress/src/utils/clampValue.test.ts

```diff
@@ -0,0 +1,43 @@
+import { clampValue } from './clampValue';
+describe('ProgressBar and value', () => {
+  const originalConsoleError = console.error;
+  beforeEach(() => {
```

Comment:
This can be `beforeAll`

```suggestion
  beforeAll(() => {
```